modules/commons/src/main/resources/schemas/neurosciencegraph/commons/annotation/v0.2.0.json

```diff
@@ -0,0 +1,202 @@
+              "name": "Selector",
+              "description": "The relationship between a Specific Resource and a Selector.",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/SelectorShape"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "TextPositionTargetShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "SelectorTargetShape"
+        },
+        {
+          "property": [
+            {
+              "path": "oa:hasSelector",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/TextPositionSelectorShape",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "FigureTargetShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "SelectorTargetShape"
+        },
+        {
+          "property": [
+            {
+              "path": "oa:hasSelector",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/FigureSelectorShape",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "EquationTargetShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "SelectorTargetShape"
+        },
+        {
+          "property": [
+            {
+              "path": "oa:hasSelector",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/EquationSelectorShape",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "TableTargetShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "SelectorTargetShape"
+        },
+        {
+          "property": [
+            {
+              "path": "oa:hasSelector",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/TableSelectorShape",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "AreaTargetShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "SelectorTargetShape"
+        },
+        {
+          "property": [
+            {
+              "path": "oa:hasSelector",
+              "node": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/AreaSelectorShape",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "BodyShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI"
+    },
```

Comment:
Is this shape necessary since there are no property shapes defined in it?

Reply:
Same as `nsg:TargetShape`.